docs/guide/http_cache.md

```diff
@@ -158,6 +158,13 @@ however in [[= product_name =]] can cover most use cases by setting supported en
     - If not set, it is automatically disabled for `APP_ENV=dev` for local development needs, otherwise enabled.
 - `TRUSTED_PROXIES`: String with trusted IP, multiple proxies can be configured with a comma, i.e. `TRUSTED_PROXIES="192.0.0.1,10.0.0.0/8"`
 
+Add the trusted proxies to your configuration:
+
+``` yaml
+framework:
+    trusted_proxies: '%env(TRUSTED_PROXIES)%'
+```
+
 !!! caution "Careful when trusting dynamic IP using TRUST_REMOTE value or similar"
```

Comment:
Also, `TRUST_REMOTE` is no longer with us, long live for `REMOTE_ADDR`.
https://github.com/ezsystems/ezplatform/pull/537